modules/shared/src/main/scala/io/constellationnetwork/currency/schema/currency.scala

```diff
@@ -162,6 +165,7 @@ object currency {
     dataApplication: Option[DataApplicationPart] = None,
     messages: Option[SortedSet[Signed[CurrencyMessage]]] = None,
     feeTransactions: Option[SortedSet[Signed[FeeTransaction]]] = None,
+    spendTransactions: Option[SortedSet[SpendTransaction]] = None,
```

Comment:
I think we need to keep spend transactions as Signed (signatures should be combined from all nodes taking part in the consensus). We would have to validate if the proper AMM was an issuer of the SpendTransaction. Especially that we want to keep it outside of the currency snapshot it belongs to (in GlobalSnapshotInfo).

Reply:
👍🏼